paasta_tools/vitesscluster_tools.py

```diff
@@ -843,6 +898,14 @@ def get_keyspaces(self) -> List[KeyspaceConfigDict]:
         labels = self.get_labels()
         node_affinity = self.get_vitess_node_affinity()
         annotations = self.get_annotations()
+        extra_volumes = self.get_kubernetes_volumes()
+        extra_volume_mounts = self.get_kubernetes_volume_mounts(extra_volumes)
+        api_client = ApiClient()
+        extra_volumes.extend(VTTABLET_EXTRA_VOLUMES)
+        for volume_mount in VTTABLET_EXTRA_VOLUME_MOUNTS:
+            extra_volume_mounts.append(
+                api_client.sanitize_for_serialization(volume_mount)
+            )
```

Comment:
should we store this in the final serialized format instead of having to convert before using them?

Reply:
This is the final format in which we expect it to be part of the spec, for reference in tests
```
{
                                            "mountPath": "/etc/vault/all_cas",
                                            "name": "vault-secrets",
                                            "readOnly": True,
                                        },
```

Reply:
Oh, do you mean to store the global variable for `VTTABLET_EXTRA_VOLUME_MOUNTS` in a serialized format instead of List[V1VolumeMount]? I did that to avoid mypy errors when performing `extra_volume_mounts.append` operation